package.json

```diff
@@ -14,11 +14,11 @@
   "dependencies": {
     "@inquirer/prompts": "^7.4.1",
     "@rhinestone/module-sdk": "^0.2.8",
-    "@rhinestone/orchestrator-sdk": "0.2.0-rc.8",
+    "@rhinestone/sdk": "^0.5.4",
     "dotenv": "^16.5.0",
     "ts-node": "^10.9.2",
     "typescript": "^5.8.3",
-    "viem": "^2.27.0"
+    "viem": "^2.28.3"
```

Comment:
The new SDK requires viem `2.28.0` (for 7702)

Reply:
we should mention this in the docs when we have them up